### DISCUSSION TITLE
Full cursor set does not appear, nor can its size be changed

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2377] [Q&A]
I have the LXQt cursor theme set to Breeze. When I double-click a desktop icon to launch an app, the circular blue and red cursor (with rotating blue and red arrows) should display until the app loads. In LXDE, this occurs. In LXQt, it does not, the cursor displayed remains the standard arrow pointer after the double-click.

Comparing the cursor settings between the two desktops, LXQt provides an option to change the size (which does not work), LXDE does not. I changed the size down to 18 and up to 24. Upon a restart of the LXQt session, the size displayed always reverted back to 21.

Could this be a bug in lxqt-config? The Debian package in sid/unstable was updated within the past day or so, from version 1.2.0-1 to 1.2.0-1+b1, but I did not attempt to change the cursor theme or size prior to this.

Thanks in advance.

**Comment by M4he** [comment]
I can confirm LXQt resetting the configured cursor size back to a smaller size on LXQt 0.16.0 on Debian 11.

**Reply by edwardp3** [reply]
Just noticed, forgot to add the distro above, which is Debian sid/unstable .

**Comment by Gnomasz** [comment]
The issue [should be resolved in lxqt-config 1.4](https://github.com/lxqt/lxqt-config/pull/938) which, unfortunately, is not even in Sid yet.

A workaround mentioned [in the bug thread](https://github.com/lxqt/lxqt/issues/2391#issuecomment-1533247900) is to "symlink ~.Xdefaults to ~/.Xresources".

A simpler workaround is to copy the contents of  ~.Xdefaults to ~/.Xresources, while removing any unnecessary copied entries, so the content looks something like this:
```
Xcursor.theme: Bibata-Original-Amber

Xcursor.size: 24
```
If you use this workaround, however, you need to manually correct it each time you want to change the cursor.

**Reply by tsujan** [reply]
> The issue should be resolved in lxqt-config 1.4

Yes, it is.
